src/main/java/org/openstreetmap/atlas/utilities/scalars/Height.java

```diff
@@ -0,0 +1,45 @@
+package org.openstreetmap.atlas.utilities.scalars;
+
+import java.io.Serializable;
+
+/**
+ * Distance wrapper to denote height above or below ground
+ *
+ * @author ajayaswal
+ */
+
+public class Height implements Serializable
+{
+    private static final long serialVersionUID = 8572975427687313909L;
+
+    private final Distance distance;
+    private final boolean aboveGround;
+
+    public Height()
+    {
+        this.distance = Distance.ZERO;
+        this.aboveGround = true;
+    }
+
+    public Height(final Double rawHeight)
```

Comment:
double instead of Double?

Reply:
Removing this class, existing Altitude class works.